components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -101,15 +110,6 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 @property(nonatomic, copy, nullable) void (^traitCollectionDidChangeBlock)
     (MDCTabBarView *_Nonnull tabBar, UITraitCollection *_Nullable previousTraitCollection);
 
-/**
- The total duration for all animations that take place during a selection change.
-
- This is guaranteed to be the total time between the start of the first animation and the end of
- the last animation that takes place for selection changes. There may not be a specific animation
- that has this exact duration.
- */
-@property(nonatomic, readonly) CFTimeInterval selectionChangeAnimationDuration;
```

Comment:
This was intentionally not an exposed constant because some expressions of Material may desire different timing details. I do expect this to be customizable in the future, but it hasn't been necessary at the moment. Rather than having a global constant that everything implicitly depends on, could you read the value from the TabBarView and pass it to custom views?  What if the API contract was amended so that it wouldn't change over the lifetime of the MDCTabBarView? In the future, it could be an initialization parameter or delegates could be called, etc.

What do you think?

Reply:
I ended up passing the value from the TabBarView instead. As we discussed offline, this is not reasonable to expose as a constant.